paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetViewModel.kt

```diff
@@ -64,7 +64,10 @@ internal class CustomerSheetViewModel @Inject constructor(
             .firstOrNull()
             ?.paymentSelection
 
-    private var originalPaymentSelection: PaymentSelection? = null
+    private var originalPaymentSelection: PaymentSelection? = backstack
+        .filterIsInstance<CustomerSheetViewState.SelectPaymentMethod>()
+        .firstOrNull()
+        ?.paymentSelection
```

Comment:
Can this ever be anything else than `null` during initialization?

Reply:
For testing purposes, yes.

Reply:
I don’t think we should add it only to make our tests pass.